components/lib/state/src/main/java/mozilla/components/lib/state/Store.kt

```diff
@@ -40,6 +44,18 @@ open class Store<S : State, A : Action>(
     private val dispatcher = Executors.newSingleThreadExecutor().asCoroutineDispatcher()
     private val scope = CoroutineScope(dispatcher)
     private val subscriptions = mutableSetOf<Subscription<S, A>>()
+    private val exceptionHandler = CoroutineExceptionHandler { _, throwable ->
+        // We want exceptions in the reducer to crash the app and get silently ignored. Therefore we rethrow the
```

Comment:
nit: and .. NOT get silently ignored? :)

Reply:
lol, yeah :)